HHS/Head-Start-TTADP
# [TTAHUB-506] Redo date picker UI

## Description of change
- Remove react-dates in favor of the react-usdws date picker.
- Remove code for previous filter implementation, since it isn't being used
- Date picker now has four options for conditions (Is after, Is before, Is within, and **in**)
- "In encompasses preset date ranges (last 30 days, etc)

## How to test
Use the date picker in the filter menu and in the activity report creation screen.
Ensure that everything works as expected.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-506


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
